saleor/discount/models.py

```diff
@@ -165,6 +166,27 @@ def validate_min_checkout_items_quantity(self, quantity):
                 min_checkout_items_quantity=min_checkout_items_quantity,
             )
 
+    def validate_once_per_customer(self, customer_email):
+        voucher_customer = VoucherCustomer.objects.filter(
+            voucher=self, customer_email=customer_email
+        )
+        if voucher_customer:
+            msg = pgettext(
+                "Voucher not applicable",
+                ("This offer is valid only once per customer."),
```

Comment:
You don't need the parentheses here.
```suggestion
                "This offer is valid only once per customer.",
```